.changeset/olive-taxis-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/contracts': minor
```

Comment:
I don't think this necessarily needs to be a minor change but not super opinionated

Reply:
Made patch instead